src/routes/(app)/committees/committee.server.ts

```diff
@@ -18,34 +18,53 @@ export type UpdateSchema = typeof updateSchema;
 
 /**
  * @param shortName The committee's short name
+ * @param year The year to load the committee for, defaults to current year
  * @returns All data that the every committee load function needs
  */
 export const committeeLoad = async (
   prisma: PrismaClient,
   shortName: string,
+  year = new Date().getFullYear(),
 ) => {
+  // const date = new Date(year, 0, 1);
```

Comment:
I think this might be a timezone thing. Be careful that this doesn't break when changing to summer time etc.

Reply:
If I create a `new Date(2024, 0, 1)` in the JS console in my browser I get jan 1st at 00:00:00, so if It was off by even 1 second it would be shunted into 2023.

Reply:
I did some changes now and hopefully they handle this better